bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -696,26 +696,73 @@ sitemap netatmo label="Netatmo" {
 }
 ```
 
+## Rule Actions
 
-# Sample data
+Multiple actions are supported by this binding. In classic rules these are accessible as shown in this example (adjust getActions with your ThingId):
+
+Example
+
+```
+ val actions = getActions("netatmo","netatmo:room:home:livingroom")
```

Comment:
@clinique : please confirm the thing UID is correct

Reply:
When generated by discovery, structure is netatmo:room:accountId:homeId:roomId

Reply:
Ok, fixed.
To be improved later in the documentation because "home" was used in the example as account id... Makes thiongs confusing.